content/docs/cml/start-gitlab.md

```diff
@@ -0,0 +1,117 @@
+# Using CML on GitLab
```

Comment:
is it the same document? a duplicate?

Reply:
Not sure what you are referring to.  Same as what.  I think I accidentally made two pull requests for the same thing.  This is the most up to date.  under branch `improve-cml-docs`.

Reply:
I see two documents here https://github.com/iterative/dvc.org/pull/2176/files: 

content/docs/cml/start-gitlab.md
content/docs/cml/get-started-with-cml-on-gitlab.md